src/types/Species.ts

```diff
@@ -20,22 +20,22 @@ export type SpeciesProblemElement = {
   suggestedValue?: string;
 };
 
-export function growthForms() {
+export function growthForms(useLocalizedValues = false) {
   return [
-    { label: strings.TREE, value: 'Tree' },
-    { label: strings.SHRUB, value: 'Shrub' },
-    { label: strings.FORB, value: 'Forb' },
-    { label: strings.GRAMINOID, value: 'Graminoid' },
-    { label: strings.FERN, value: 'Fern' },
+    { label: strings.TREE, value: useLocalizedValues ? strings.TREE : 'Tree' },
```

Comment:
Don't we need to translate the label instead of the value? Because I tried editing a species and then saving (in gibberish mode) and i get an api error {"error":{"message":"Field value invalid: growthForm"},"status":"error"}

Reply:
You mean you tried with this version of the code, or with main? That bug should be fixed by this PR.

Reply:
With this version, in Vercel

Reply:
Ah, oops, the fix was actually a backend thing and I neglected to send out a PR for it.

Reply:
PR is out now: https://github.com/terraware/terraware-server/pull/930